src/config.ts

```diff
@@ -0,0 +1,13 @@
+let moment = require('moment');
+
+export class Config {
```

Comment:
You could probably export a singleton object instead of a class?

Reply:
I was hoping for something more scala-like that any of these - https://stackoverflow.com/questions/30174078/how-to-define-singleton-in-typescript